src/plugins/intel_cpu/src/nodes/reduce.cpp

```diff
@@ -3133,6 +3185,7 @@ std::vector<int> Reduce::update_src_dims() {
 bool Reduce::canApplyJIT(const Precision &input_prec, const Precision &output_prec) const {
     static const Precision supportedPrecisions[] = {
             Precision::FP32,
+            Precision::FP16,
```

Comment:
Good point! Applied. I've changed the logic after this function to make sure avx2 is available for f16, otherwise the precision supported will be reset to fp32. Thanks Chenhu!